docs/howto/account-billing/Delete-account.md

```diff
@@ -0,0 +1,24 @@
+# Deleting your account
+
+In case you decide to close your account in {{brand}}, you may do so via the [{{gui}}](https://{{gui_domain}}).
+Log into your account and click on the profile button at the top right.
+
+![Orange avatar person with white background.](assets/picture-profile.png)
+
+A pane named _Account settings_ will slide over from the right-hand side of the browser window. To delete your account, click the _Settings_ tab.
+
+![A Settings tab with a red box around it](assets/accountsettings.png)
+
+Scroll to the bottom of the pane, and click the red button named _Delete Account_.
+
+![Black text on white background with info what will happen when you click on the delete button](assets/Delete-account-picture.png)
+
+Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_. That will create a request for account deletion.
+
+![Picture where you choose a option why you leave us. A field where you type your current password with a red delete button.](assets/reasonforleaving.png)
+
+Three (3) days after sending the request for deletion, any running virtual machines will be shut off. Then, after ten (10) days, all virtual machines and any other resources connected to your account will be deleted, **including** backups.
+
+After the last invoice has been paid, we will delete your account. This entire process may take up to thirty (30) days after the last invoice is paid.
+
+If you change your mind during the first 3 days please contact [Service Center](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** any of your assets, nor your account.
```

Comment:
To be consistent with how we specify the time (in days), I'm now suggesting this:

```suggestion
If you change your mind during the first three (3) days, please get in touch with [Service Center](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** your assets or your account.
```

I also went with "get in touch", for I find it friendlier than "contact" :)